src/components/dropdown/dropdown.e2e.ts

```diff
@@ -1024,4 +1026,56 @@ describe("calcite-dropdown", () => {
         shadowPopperSelector: ".calcite-dropdown-wrapper"
       }
     ));
+
+  it("should emit component status for transition-chained events: 'calciteDropdownBeforeOpen', 'calciteDropdownOpen', 'calciteDropdownBeforeClose', 'calciteDropdownClose'", async () => {
+    const page = await newE2EPage();
+    await page.setContent(html`
+      <calcite-dropdown>
+        <calcite-button slot="dropdown-trigger">Open dropdown</calcite-button>
+        <calcite-dropdown-group id="group-1">
+          <calcite-dropdown-item id="item-1"> Dropdown Item Content </calcite-dropdown-item>
+          <calcite-dropdown-item id="item-2" active> Dropdown Item Content </calcite-dropdown-item>
+          <calcite-dropdown-item id="item-3"> Dropdown Item Content </calcite-dropdown-item>
+        </calcite-dropdown-group>
+      </calcite-dropdown>
+    `);
```

Comment:
Similarly here, you can simplify this with `await newE2EPage({ html: <html-goes-here> })`.

Reply:
I believe setting page through `page.setContent` was @jcfranco's preference, as he corrected it a couple of times. From what I understand as opposed to `await newE2EPage()`, `page.setContent()` should have `await page.waitForChanges()` built in. I might be wrong though.

Reply:
Lol, I recall Franco doing the exact opposite on some of my PRs where he recommended setting it the way I described above.  I honestly don't have a preference either way, just was following what I know I've encountered on reviews in the past.

Reply:
Ok :) Maybe we should convene at some point and pick one :).

Reply:
I put it on the agenda for the next time to agree on a preferred method of setting the page content.

Reply:
Fine to leave as-is for now, shouldn't block merging.